app/views/deductionBreakdown.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+ @import models.calculation.CalcDisplayModel
+ @import views.html.templates.main_template
+ @import implicits.ImplicitCurrencyFormatter._
+ @import views.html.helpers.backLinkHelper
+
+@(calcModel: CalcDisplayModel, taxYear: Int)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig, user: auth.MtdItUser[_])
+
+@tableRow(messageCode: String, dataItem: Option[BigDecimal]) = {
+  @if(dataItem.isDefined) {
+   <tr id="@messageCode">
+    <td>@messages(s"deduction_breakdown.table.$messageCode")</td>
+    <td class="numeric">@{dataItem.get.toCurrency}</td>
+   </tr>
+  }
+}
+
+@main_template(messages("deduction_breakdown.heading",  s"${taxYear-1}", s"$taxYear"),
+ contentHeader = Some(backLinkHelper(controllers.routes.CalculationController.renderCalculationPage(taxYear).url))
+) {
+
+ <h1 class="heading-xlarge">
+  <span class="heading-secondary">@messages("deduction_breakdown.dates", s"${taxYear-1}", s"$taxYear")</span>
+  @messages("deduction_breakdown.heading")
+ </h1>
+
+ <div>
+  <p>
+   @messages("deduction_breakdown.guidance_software", s"${taxYear-1}", s"$taxYear")
+   <a href="https://www.gov.uk/guidance/find-software-thats-compatible-with-making-tax-digital-for-income-tax" target="_blank">@messages("deduction_breakdown.guidance_software.link") <span class="visually-hidden">@messages("obligations.opens-in-new-tab")</span></a>.
+  </p>
+
+  <table>
+   @tableRow("personal_allowance", calcModel.calcDataModel.allowancesAndDeductions.personalAllowance)
+   @tableRow("gift_of_investments_and_property_to_charity", calcModel.calcDataModel.allowancesAndDeductions.giftOfInvestmentsAndPropertyToCharity)
+   <tr class="no-border-bottom">
+    <td class="total-section">@messages("deduction_breakdown.total")</td>
+    <td class="numeric total-section">@calcModel.calcDataModel.allowancesAndDeductions.totalAllowancesAndDeductions.getOrElse(BigDecimal(0)).toCurrency</td>
```

Comment:
The value used on the overview page is calculation.allowancesAndDeductions.totalAllowancesDeductionsReliefs from the model, the value used on the overview and the value you have used might not match up.